src/PointwiseFunctions/AnalyticSolutions/Elasticity/BentBeam.hpp

```diff
@@ -47,18 +47,24 @@ namespace Solutions {
  * By fixing the rigid-body motions to
  *
  * \f[
- * u^x(0,y)=0 \quad \text{and} \quad u^y\left(\pm \frac{L}{2},0\right)=0
+ * \xi^x(0,y)=0 \quad \text{and} \quad \xi^y\left(\pm \frac{L}{2},0\right)=0
```

Comment:
Please cite an equation and reference for this solution. I think it’s in Thorne and Blandford?

Reply:
Blandford, Thorne only discusses the 1D-case. @nilsleiffischer derived the solution himself